android/app/build.gradle

```diff
@@ -57,19 +57,19 @@ android {
             // TODO: Add your own signing config for the release build.
             // Signing with the debug keys for now, so `flutter run --release` works.
             signingConfig signingConfigs.debug
-            applicationVariants.all { variant ->
-                variant.outputs.all {
-                    def appName = "threedpass_"
-                    def buildType = variant.variantData.variantConfiguration.buildType.name
-                    def newName
-                    if (buildType == 'debug'){
-                        newName = "app-debug.apk"
-                    } else {
-                        newName = "${appName}${defaultConfig.versionName}.apk"
-                    }
-                    outputFileName = newName
-                }
-            }
+            // applicationVariants.all { variant ->
```

Comment:
Can we remove comments right now or they can be useful later?

Reply:
I commented out the code just to get rid of the compilation error. There's some useful information left here